lib/common.js

```diff
@@ -33,6 +33,7 @@ common.safeRequire = (moduleName) => {
   try {
     return require(moduleName);
   } catch (err) {
+    console.warn(err.toString());
```

Comment:
In future we will redirect this to logging provider. Just don't forget. Better create issue :)
For example in Impress to `application.log.warning`, if no provider then print to stdout.